manifests/server/instance/config.pp

```diff
@@ -245,58 +245,11 @@
     }
   }
   # lint:ignore:140chars
-  # RHEL 7 and 8 both support drop-in files for systemd units.  The old include directive is deprecated and may be removed in future systemd releases.
+  # RHEL 7 and 8 both support drop-in files for systemd units.
   # Gentoo also supports drop-in files.
+  # Edit 02/2023 Lots of Linuxes use Systemd this time, do we want to use this on maybe all supported Systems?
```

Comment:
@chelnak @bastelfreak came to my mind, I left a comment

Reply:
I think it depends on how the other systems use overrides. For example, on Debian it doesn't appear to support changing the data directory. That always uses `/var/lib/postgresql/%I` so I'd say there is no point in doing so. The same for changing ports.

In fact, I would check if changing the data directory and port is still up to date.

For example, changing the data directory via `postgresql.conf` is actually mostly invalid. In https://bugzilla.redhat.com/show_bug.cgi?id=1935301 I was told this breaks various tooling. In upstream (https://www.postgresql.org/message-id/3566642.1618422939%40sss.pgh.pa.us)  they recommended filtering it out of `postgresql.conf`.

The main reason is that the config (at least on RHEL) is present in the data directory, so you end up with a chicken-and-egg situation.

Reply:
I checked this on a RHEL 7 with SCL Postgres 13, the port now works out of the box, the data directory ist still needed

Reply:
Edited the comment